cmd/create/cmd.go

```diff
@@ -24,14 +24,16 @@ import (
 	"github.com/openshift/moactl/cmd/create/idp"
 	"github.com/openshift/moactl/cmd/create/ingress"
 	"github.com/openshift/moactl/cmd/create/user"
+	v "github.com/openshift/moactl/cmd/validations"
 	"github.com/openshift/moactl/pkg/interactive"
 )
 
 var Cmd = &cobra.Command{
-	Use:     "create RESOURCE [flags]",
-	Aliases: []string{"add"},
-	Short:   "Create a resource from stdin",
-	Long:    "Create a resource from stdin",
+	Use:              "create RESOURCE [flags]",
+	Aliases:          []string{"add"},
+	Short:            "Create a resource from stdin",
+	Long:             "Create a resource from stdin",
+	PersistentPreRun: v.Validations,
```

Comment:
This should probably live only under `create cluster`, putting it here makes it run for all resources (e.g. `moactl create user`) and these operations do not need to validate the stack, as we know it already exists (due to the existence of the cluster itself)

Reply:
Addressed and resolved